feature/gnmi/ate_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -96,49 +97,58 @@ func TestInterfaceCounters(t *testing.T) {
 	ipv4CounterPath := "/interfaces/interface/subinterfaces/subinterface/ipv4/state/counters/"
 	ipv6CounterPath := "/interfaces/interface/subinterfaces/subinterface/ipv6/state/counters/"
 
-	cases := []struct {
+	type counterCase struct {
 		desc    string
 		path    string
-		counter *telemetry.QualifiedUint64
-	}{{
+		counter func(testing.TB) *telemetry.QualifiedUint64
+	}
+	type counterCases []counterCase
+	cases := make(counterCases, 0)
+	cases = append(cases, counterCase{
 		desc:    "InUnicastPkts",
 		path:    intfCounterPath + "in-unicast-pkts",
```

Comment:
Can we use only variable cases instead of 2?

Reply:
"cases" is variable of type counterCases. This is required since only typed slice variables can be used with "append" function.